gflownet/gflownet.py

```diff
@@ -475,12 +460,27 @@ def sample_batch(
                 dict_replay = pickle.load(f)
                 n_replay = min(n_replay, len(dict_replay["x"]))
                 envs = [self.env.copy().reset(idx) for idx in range(n_replay)]
-                # TODO: implement other sampling options besides permutation
```

Comment:
Note to self (to be directly resolved): new code about replay options looks good to me.